src/cached_tokenizers.rs

```diff
@@ -150,6 +150,7 @@ pub async fn cached_tokenizer(
     try_download_tokenizer_file_and_open(&client2, http_path.as_str(), api_key.clone(), &to).await?;
     info!("loading tokenizer \"{}\"", to.display());
     let mut tokenizer = Tokenizer::from_file(to).map_err(|e| format!("failed to load tokenizer: {}", e))?;
+    let _ = tokenizer.with_truncation(None);
```

Comment:
"The with_truncation function in the context of tokenizers is used to enable or disable truncation of the tokenized sequence. When truncation is enabled, sequences longer than a specified maximum length will be truncated to fit within that limit. The None parameter likely indicates that there is no specific truncation configuration provided, meaning default settings might be used."
what does it really mean?